src/controller/python/chip/clusters/Command.py

```diff
@@ -201,6 +283,61 @@ def SendCommand(future: Future, eventLoop, responseType: Type, device, commandPa
         ))
 
 
+def SendBatchCommands(future: Future, eventLoop, device, commands: List[Tuple[int, ClusterCommand, Type]],
+                      timedRequestTimeoutMs: Union[None, int] = None, interactionTimeoutMs: Union[None, int] = None, busyWaitMs: Union[None, int] = None,
+                      suppressResponse: Union[None, bool] = None) -> PyChipError:
+    ''' Send a cluster-object encapsulated command to a device and does the following:
+            - On receipt of a successful data response, returns the cluster-object equivalent through the provided future.
+            - None (on a successful response containing no data)
+            - Raises an exception if any errors are encountered.
+
+        If no response type is provided above, the type will be automatically deduced.
+
+        If a valid timedRequestTimeoutMs is provided, a timed interaction will be initiated instead.
+        If a valid interactionTimeoutMs is provided, the interaction will terminate with a CHIP_ERROR_TIMEOUT if a response
+        has not been received within that timeout. If it isn't provided, a sensible value will be automatically computed that
+        accounts for the underlying characteristics of both the transport and the responsiveness of the receiver.
+    '''
+    handle = chip.native.GetLibraryHandle()
+
+    responseTypes = []
+    commandargs = []
+    for command in commands:
+        endpoint = command[0]
```

Comment:
Check that `len(command) >= 3` prior to indexing. Also, check that it's a Tuple (`isinstance(command, tuple)`) since that would allow having multiple value types provided over time (e.g. Tuple like now, and a future CommandRequest dataclass or such)